Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM opensearchproject/opensearch:2.14.0
+FROM opensearchstaging/opensearch:3.0.0
```

Comment:
is this kind of the pattern that plugins use?

Reply:
Yes, it seems to be.